lib/assessment.js

```diff
@@ -24,7 +33,9 @@ var sum = function sum() {
 // if invoking the callback for any element
 // in the array returns a truthy value
 var any = function any(array, callback, thisArg) {
-
+  var ifAny = false;
+  //for each element in the array, use callback, if callback returns true set ifAny to true
+  return ifAny;
```

Comment:
I'll give this 1/4 credit for correct pseudocode and a good starting value for `ifAny`.